tests/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# This file is a part of IoT-LAB gateway_code
+# Copyright (C) 2015 INRIA (Contact: admin@iot-lab.info)
+# Contributor(s) : see AUTHORS file
+#
+# This software is governed by the CeCILL license under French law
+# and abiding by the rules of distribution of free software.  You can  use,
+# modify and/ or redistribute the software under the terms of the CeCILL
+# license as circulated by CEA, CNRS and INRIA at the following URL
+# http://www.cecill.info.
+#
+# As a counterpart to the access to the source code and  rights to copy,
+# modify and redistribute granted by the license, users are provided only
+# with a limited warranty  and the software's author,  the holder of the
+# economic rights,  and the successive licensors  have only  limited
+# liability.
+#
+# The fact that you are presently reading this means that you have had
+# knowledge of the CeCILL license and that you accept its terms.
+
+FROM iot-lab-gateway
+
+# Update
+RUN apt-get update
+RUN apt-get install -y python-pip
+
+# Install app dependencies
+RUN pip install --upgrade pip
+
+RUN pip install tox
+
+CMD [ "tox","--develop" ]
```

Comment:
Missing newline (not sure this is very important though but GitHub reports this)